### PR TITLE
Fix peer and Create Audit

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,6 @@
     "kill": "pnpm run kill --filter {'apps'}"
   },
   "devDependencies": {
-    "turbo": "latest"
+    "turbo": "^1.3.1"
```
Comment (ogroppo) [inline package.json:16]: did this cause issues?

**Review by ogroppo — APPROVED** [review]

File: packages/ra-data-simple-prisma/src/defaultHandler.ts [hunk 2]
```diff
@@ -62,6 +63,7 @@ export const defaultHandler = async (
     }
     case "create": {
       return await createHandler(req as CreateRequest, res, model, {
+        ...options?.create,
```
Comment (ogroppo) [inline packages/ra-data-simple-prisma/src/defaultHandler.ts:66]: does this work if options is undefined? 🤔

**Review by ogroppo — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (martin-fv) [reply package.json:16]: no, just was set by pnpm upgrade

File: packages/ra-data-simple-prisma/src/defaultHandler.ts (hunk 2 above)
Comment (martin-fv) [reply packages/ra-data-simple-prisma/src/defaultHandler.ts:66]: yup

**Pull request merged**